test/minifiedEquippable.ts

```diff
@@ -0,0 +1,250 @@
+// Note: This is just a copy of the equippable test suit with the additional tests
+// for ERC721 and nestable behavior, but using MinifiedEquippable instead.
+
+import { Contract } from 'ethers';
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
+import {
+  addAssetEntryEquippablesFromMock,
+  addAssetToToken,
+  mintFromMock,
+  nestMintFromMock,
+  nestTransfer,
+  parentChildFixtureWithArgs,
+  transfer,
+} from './utils';
+import { setupContextForParts } from './setup/equippableParts';
+import { setupContextForSlots } from './setup/equippableSlots';
+import shouldBehaveLikeEquippableAssets from './behavior/equippableAssets';
+import shouldBehaveLikeEquippableWithParts from './behavior/equippableParts';
+import shouldBehaveLikeEquippableWithSlots from './behavior/equippableSlots';
+import shouldBehaveLikeNestable from './behavior/nestable';
+import shouldBehaveLikeMultiAsset from './behavior/multiasset';
+import shouldBehaveLikeERC721 from './behavior/erc721';
+import {
+  RMRKCatalogMock,
+  RMRKMinifiedEquippableMock,
+  RMRKEquipRenderUtils,
+  RMRKMultiAssetRenderUtils,
+} from '../typechain-types';
+// --------------- FIXTURES -----------------------
+
+async function partsFixture() {
+  const catalogSymbol = 'NCC';
+  const catalogType = 'mixed';
+
+  const neonName = 'NeonCrisis';
+  const neonSymbol = 'NC';
+
+  const maskName = 'NeonMask';
+  const maskSymbol = 'NM';
+
+  const catalogFactory = await ethers.getContractFactory('RMRKCatalogMock');
+  const equipFactory = await ethers.getContractFactory('RMRKMinifiedEquippableMock');
+  const viewFactory = await ethers.getContractFactory('RMRKEquipRenderUtils');
+
+  // Catalog
+  const catalog = <RMRKCatalogMock>await catalogFactory.deploy(catalogSymbol, catalogType);
+  await catalog.deployed();
+
+  // Neon token
+  const neon = <RMRKMinifiedEquippableMock>await equipFactory.deploy(neonName, neonSymbol);
+  await neon.deployed();
+
+  // Weapon
+  const mask = <RMRKMinifiedEquippableMock>await equipFactory.deploy(maskName, maskSymbol);
+  await mask.deployed();
+
+  // View
+  const view = <RMRKEquipRenderUtils>await viewFactory.deploy();
+  await view.deployed();
+
+  await setupContextForParts(catalog, neon, neon, mask, mask, mintFromMock, nestMintFromMock);
+  return { catalog, neon, mask, view };
+}
+
+async function slotsFixture() {
+  const catalogSymbol = 'SSB';
```

Comment:
This is my final suggestion:

```suggestion
  const catalogSymbol = 'SSC';
```